core/cmd/local_client.go

```diff
@@ -439,6 +439,32 @@ func (cli *Client) DeleteUser(c *clipkg.Context) (err error) {
 	return err
 }
 
+// SetNextNonce manually updates the keys.next_nonce field for the given key with the given nonce value
+func (cli *Client) SetNextNonce(c *clipkg.Context) error {
+	addressHex := c.String("address")
+	nextNonce := c.Uint64("nextNonce")
+
+	logger.SetLogger(cli.Config.CreateProductionLogger())
+	db, err := gorm.Open("postgres", cli.Config.DatabaseURL())
```

Comment:
The local client connects to the database :o

Reply:
It could be run while the node is running, although it is probably safer if the node is not running.

Nonce does not exist in memory the source of truth is always the database.

I will update to use `orm.DialectPostgresWithoutLock` :+1: